.github/workflows/latest.yaml

```diff
@@ -35,8 +35,8 @@ jobs:
     - name: Lint
       uses: golangci/golangci-lint-action@v2
       with:
-        version: v1.32.2
-        args: --timeout=3m
+        version: v1.38.0
+        args: -p bugs --timeout=3m
```

Comment:
Maybe we can use some linters that are not in `bugs` preset, like `deadcode` and `unused`. For that we will need additional config file for golangci-lint. Probably should be done in separate issue.

Reply:
I would prefer not to create a golangci-lint config because this is one more file to manage, having the cli options in th CI/CD is more obvious.

Reply:
BTW: version can be removed, then the most recent version is used which is what we do now in all other projects step by step.